calculate.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require_once 'database/database.php';
+
+$wyniki = $pdo->query('SELECT * FROM leki')->fetchAll(PDO::FETCH_ASSOC);
+
+foreach ($wyniki as $wynik)
+
+    {
+        $dzisiejsza_data = date("Y-m-d");
+        $data_dodania = $wynik['Data'];
+        $zapotrzebowanie = $wynik['Zapotrzebowanie'];
+
+        $datetime1 = new DateTime($data_dodania);
```

Comment:
Możesz od razu przekazać $wynik['Data'] tutaj, to samo ponizej z dzisiejsza data